code/game/gamemodes/colonialmarines/main.dm

```diff
@@ -40,6 +40,11 @@
 		if(islarva(M.current))
 			var/mob/living/carbon/alien/larva/L = M.current
 			L.amount_grown = L.max_grown
+
+			for (var/v = 1, v <= 3, v++)
+				if (prob(50))
+					new/mob/living/carbon/monkey(locate(L.x + rand(-1, 1), L.y + rand(-1, 1), L.z))
```

Comment:
locate might be a bit laggy,you sure there's no other way to do this?i think locate does an "in world" check..